pkgs/applications/audio/diopser/default.nix

```
@@ -0,0 +1,74 @@
+{ lib, stdenv, fetchFromGitHub, cmake, pkg-config
+, libjack2, alsaLib, freetype, libX11, libXrandr, libXinerama, libXext, libXcursor
+}:
+
+let
+
+  # Derived from subprojects/function2.wrap
+  function2 = rec {
+    version = "4.1.0";
+    src = fetchFromGitHub {
+      owner = "Naios";
+      repo = "function2";
+      rev = version;
+      hash = "sha256-JceZU8ZvtYhFheh8BjMvjjZty4hcYxHEK+IIo5X4eSk=";
+    };
+  };
+
+  juce = rec {
+    version = "1a5fb5992a1a4e28e998708ed8dce2cc864a30d7";
```

```suggestion
  juce = {
    version = "unstable-XXXX-XX-XX";
```

replace XXXX-XX-XX with the date of the commit.